dust_extinction/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = 'dust_extinction' %}
+{% set version = '0.7' %}
+{% set number = '0' %}
+
+about:
+    home: https://github.com/karllark/{{ name }}
+    license: BSD
+    summary: |
+        Provides models of interstellar dust extinction curves
+
+build:
+    number: {{ number }}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+requirements:
+    build:
+    - astropy >=1.3
+    - pytest
+    - setuptools
+    - numpy {{ numpy }}
```

Comment:
I don't see "numpy" and "python" variables defined in this file.

Reply:
Those jinja2 vars are defined by the build system

Reply:
Yeah, I just copied another astropy affiliated package, so assumed this would work.